app/controllers/admin/api_docs/account_api_docs_controller.rb

```diff
@@ -30,4 +31,8 @@ def service
   def current_scope
     current_account
   end
+
+  def find_service_apis_collection
+    @service_apis = current_account.services.accessible
```

Comment:
Would be good to define `services.accessible` in Account what do you think?

Reply:
Not that you mention it, I think it even already exists 🤔 Will check, but if it doesn't, I will create it then 😉